bot.go

```diff
@@ -56,6 +59,13 @@ func (x *opBot) Run() {
 			}
 
 		case update.Message != nil:
+			// Stats.
+			if update.Message.From != nil && update.Message.From.UserName == osProgramadoresUserName {
+				if saved, err := saveStats(&update); err != nil {
+					log.Println(T("stats_error_saving"), err.Error(), saved)
```

Comment:
*Maybe* use log.Printf here and add the %s to the error message? This will allow us to control where err.Error() and saved are printed. (May be required for some language).